scripts/__input_class_cursor/__input_class_cursor.gml

```diff
@@ -101,6 +102,72 @@ function __input_class_cursor() constructor
+            if (_gamepad_index >= 0)
+            {
+                _motion = global.__input_gamepads[_gamepad_index].__motion;
+            }
+            
+            if ((_motion != undefined) && __player.__gyro_enabled)
+            {
+                var _motion_data = _motion.__tick();
+
+                var _gyro_value_x = undefined;
+                switch (__player.__gyro_axis_x)
+                {
+                    case INPUT_GYRO.AXIS_PITCH: _gyro_value_x = _motion_data.angular_velocity_x; break;
+                    case INPUT_GYRO.AXIS_YAW:   _gyro_value_x = _motion_data.angular_velocity_y; break;
+                    case INPUT_GYRO.AXIS_ROLL:  _gyro_value_x = _motion_data.angular_velocity_z; break;
+                }
+
+                var _gyro_value_y = undefined;
+                switch (__player.__gyro_axis_y)
+                {
+                    case INPUT_GYRO.AXIS_PITCH: _gyro_value_y = _motion_data.angular_velocity_x; break;
+                    case INPUT_GYRO.AXIS_YAW:   _gyro_value_y = _motion_data.angular_velocity_y; break;
+                    case INPUT_GYRO.AXIS_ROLL:  _gyro_value_y = _motion_data.angular_velocity_z; break;
+                }
+
+                var _screen_width  = display_get_width();
+                var _screen_height = display_get_height();
+                switch(global.__input_pointer_coord_space)
+                {
+                    case INPUT_COORD_SPACE.ROOM:
+                        if (view_enabled && view_visible[0])
+                        {
+                            var _camera = view_camera[0];
+                            _screen_width  = camera_get_view_width(_camera);
+                            _screen_height = camera_get_view_height(_camera);
+                        }
+                        else
+                        {
+                            _screen_width  = room_width;
+                            _screen_height = room_height;
+                        }
+                    break;
+                
+                    case INPUT_COORD_SPACE.GUI:                        
+                        _screen_width  = display_get_gui_width();
+                        _screen_height = display_get_gui_height();
+                    break;
+                
+                    case INPUT_COORD_SPACE.DISPLAY:
+                        if (!__INPUT_ON_CONSOLE && (window_get_width != undefined))
+                        {
+                            _screen_width  = window_get_width();
+                            _screen_height = window_get_height();
+                        }
+                    break;
+                }
+
+                var _dts = delta_time/1000000;
+                if (_gyro_value_x != undefined) __x += round((_gyro_value_x * _dts * _screen_width  * __player.__gyro_sensitivity_x * 10)) / 10;
+                if (_gyro_value_y != undefined) __y += round((_gyro_value_y * _dts * _screen_height * __player.__gyro_sensitivity_y * 10)) / 10;
```

Comment:
What's the `*10` `/10` doing here?

Reply:
Resolution. `round(x*10)/10)`, so increments of 0.1 Can add a clarifying comment.